src/pages/dashboardOverview/DashboardOverview.tsx

```diff
@@ -34,14 +34,24 @@ const DashboardOverview = ({ party, products }: Props) => {
     [party.products]
   ) ?? [party.products];
 
-  const delegableProducts = activeProducts.find(
-    (p) => p.productId === 'prod-io' || p.productId === 'prod-pagopa'
-  );
+  const hasAdminRoleForSpecificProducts = (activeProducts: Array<OnboardedProduct>): boolean => {
+    const targetProducts = ['prod-io', 'prod-pagopa'];
+
+    return activeProducts.some(
+      (product: OnboardedProduct) =>
+        targetProducts.includes(product.productId ?? '') &&
+        product.userRole?.toLowerCase() === 'admin'
+    );
+  };
+
+  const isDelegable = ENV.DELEGATIONS.ENABLE && isLoginAsAdmin;
 
-  const isDelegable = ENV.DELEGATIONS.ENABLE && isAdmin;
+  const institutionsThatCanDelegate = ['PA', 'PSP', 'GSP', 'SCP'];

```

Comment:
The only one institutionType that can't delegate is PT, correct? so you can do party.institutionType !== 'PT'